assets/css/style.css

```diff
@@ -0,0 +1,663 @@
+  }
+
+  .about .info,
+  .about .casualose {
+    width: 100%;
+  }
+
+  .about .casualose {
+    padding: 0;
+  }
+
+  .footer {
+    padding: 102px 12px 48px 24px;
+  }
+
+  .footer .desc {
+    width: auto;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    padding-bottom: 32px;
+  }
+
+  .footer form {
+    padding: 0 3rem 0 0;
+  }
+
+  .footer .contact-button {
+    flex-direction: row;
+    width: 100%;
+    padding-left: 0.5rem;
+  }
+
+  .footer .button-outline {
+    justify-content: flex-start;
+    flex-flow: column;
+  }
+
+  input,
+  textarea {
+    width: 100%;
+  }
+}
+
+@media all and (min-width: 768px) {
+  .menu {
+    align-items: flex-start;
+    flex-wrap: nowrap;
+  }
+
+  .logo {
+    order: 0;
+  }
+
+  .item {
+    order: 1;
+    position: relative;
+    display: block;
+    width: auto;
+    padding-top: 10px;
+    padding-bottom: 10px;
+  }
+
+  .mobile-menu {
+    display: none;
+  }
+}
+
+@media all and (min-width: 1024px) {
+  .is_mobile {
+    display: none;
+  }
+
+  .is_desktop {
+    display: block;
+  }
+
+  .heading::before {
+    background-image: url("../images/shape_desktop.svg");
+  }
+
+  .heading {
+    padding: 8rem;
+  }
+
+  nav {
+    padding-right: 155px;
+    padding-left: 100px;
+  }
+
+  .heading__title {
+    font-size: 48px;
+    line-height: 60px;
+  }
+
+  .heading__desc {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .tonic {
+    margin: 2rem;
+    padding: 1.5rem;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .content {
+    padding: 24px;
+  }
+
+  .content .title {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .content .desc {
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .content .summary {
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .about {
+    padding-bottom: 8rem;
+    grid-template-columns: 1fr 1fr;
+  }
+
+  .about .info {
+    width: 100%;
+  }
+
+  .about .casualose {
+    width: 100%;
+    padding: 0 2rem;
+  }
+
+  .about .casualose .language {
+    display: flex;
+    -moz-column-gap: 12px;
+    column-gap: 12px;
+  }
+
+  .about .casualose .language li {
+    flex-direction: column;
+    align-items: flex-start;
+    flex-basis: 6.6rem;
+  }
+
+  .footer::before {
+    background-image: url("../images/footer_desktop.svg");
+    background-size: cover;
+    background-position: center;
+    top: 0;
+    left: 0;
+    content: "";
+    position: absolute;
+    background-repeat: no-repeat;
+    width: 100%;
+    height: 100%;
+    z-index: -1;
+  }
+
+  .footer {
+    padding: 8rem 5rem;
+  }
+
+  .footer .contact-button {
+    flex-direction: row;
+    display: flex;
+    justify-content: center;
+  }
+
+  .footer .desc {
+    width: 569px;
+  }
+
+  .footer form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    padding: 0;
+  }
+
+  input,
+  textarea {
+    width: 447px;
+  }
+}
+
+@media all and (min-width: 1440px) {
```

Comment:
- **[OPTIONAL]** There are some differences between your design and the Figma design. As requested by the previous CR it would have been enough to mark one breakpoint at width 768px for the desktop version 👉🏽 **@media screen and (min-width: 750px)** It will save you time to have two different styles one for the mobile version and the other for versions higher than 768px width.